src/store/model/bestbuy-ca.ts

```diff
@@ -3,8 +3,8 @@ import {Store} from './store';
 export const BestBuyCa: Store = {
 	labels: {
 		inStock: {
-			container: '.x-product-detail-page .addToCartButton:enabled',
-			text: ['Add to Cart']
+			container: '.pricingContainer_25k3c',
+			text: ['href="http://schema.org/InStock"']
```

Comment:
Looking into href="http://schema.org/InStock" it seems there are other valid states when look at different products on bestbuy.ca [https://schema.org/ItemAvailability](https://schema.org/ItemAvailability) such as "http://schema.org/OnlineOnly"

Reply:
I'm not certain but is it possible to have more than one `text: [ ]` to search for?  Is it possible to just comma separate them and add more?